tests/python/test_loop_grad.py

```diff
@@ -27,9 +27,8 @@ def func():
     func.grad()
 
     for k in range(n):
-        for i in range(m):
-            assert x[k, i] == 2**i * k
-            assert x.grad[k, i] == 2**(m - 1 - i)
+        assert x[k, 0] == 2**0 * k
+        assert x.grad[k, 0] == 2**(m - 1 - 0)
```

Comment:
`2 ** 0 *k` & `2**(m - 1- 0)` are weird expressions to show up here :P 
If you write it in this form, it'd be nice to have a comment explaining why ;)

Reply:
Sure. Have added some comments.